components/integrations/bots/add_bot/add_bot.jsx

```diff
@@ -0,0 +1,516 @@
+                                htmlFor='boticon'
+                            >
+                                <FormattedMessage
+                                    id='bots.add.icon'
+                                    defaultMessage='Bot Icon'
+                                />
+                            </label>
+                            <div className='col-md-5 col-sm-8'>
+                                <div className={'bot-img-container'}>
+                                    <img
+                                        className={'bot-img'}
+                                        alt={'bot image'}
+                                        src={imageURL}
+                                        style={imageStyles}
+                                    />
+                                    {removeImageIcon}
+                                </div>
+                                <div
+                                    className='btn btn-sm btn-primary btn-file sel-btn'
+                                >
+                                    <FormattedMessage
+                                        id='setting_picture.select'
+                                        defaultMessage='Select'
+                                    />
+                                    <input
+                                        accept='.jpg,.png,.bmp'
+                                        type='file'
+                                        onChange={this.updatePicture}
+                                    />
+                                </div>
+                            </div>
+                        </div>
+                        <div className='form-group'>
+                            <label
+                                className='control-label col-sm-4'
+                                htmlFor='displayName'
+                            >
+                                <FormattedMessage
+                                    id='bots.add.displayName'
+                                    defaultMessage='Display Name'
+                                />
+                            </label>
+                            <div className='col-md-5 col-sm-8'>
+                                <input
+                                    id='displayName'
+                                    type='text'
+                                    maxLength='64'
+                                    className='form-control'
+                                    value={this.state.displayName}
+                                    onChange={this.updateDisplayName}
+                                />
+                                <div className='form__help'>
+                                    <FormattedMessage
+                                        id='bot.add.display_name.help'
+                                        defaultMessage={'You can choose to display your bot\'s full name rather than its username.'}
```

Comment:
```suggestion
                                        defaultMessage={'(Optional) You can choose to display your bot\'s full name rather than its username.'}
```

Reply:
I'm not sure if this is an optional field in the backend, given creating bots throws a 404 error

Reply:
Ya. It's optional.